### PR TITLE
Initial commit.

### PR DESCRIPTION
**Pull request by mikegfink**
Test comment

### CODE REVIEW

**Comment by mikegfink** [comment]
Comment after creating pull request.

File: main.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// ==UserScript==
+// @name         ReviewHub
+// @namespace
+// @version      0.1
+// @description  Modify GitHub pull requests to require approval
```
Comment (mikegfink) [inline main.js]: Test file comment.

File: main.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+// @author       Mike Fink
+// @match        https://github.com/*/*/pull/*/files
+// @require      http://code.jquery.com/jquery-2.2.1.min.js
+// ==/UserScript==
+/* jshint -W097 */
+'use strict';
+
```
Comment (mikegfink) [inline main.js]: Second test comment

**Comment by mikegfink** [comment]
Second commit comment

**Comment by mikegfink** [comment]
:+1:

**Comment by mikegfink** [comment]
@mikegfink

**Comment by mikegfink** [comment]
@nicviclee%

**Comment by nicviclee** [comment]
:shipit:

**Comment by mikegfink** [comment]
test

**Comment by mikegfink** [comment]
testagain

**Comment by mikegfink** [comment]
adding a new comment

**Comment by mikegfink** [comment]
test

**Comment by mikegfink** [comment]
@mikegfink%

**Comment by mikegfink** [comment]
@mikegfink%

**Comment by mikegfink** [comment]
@mikegfink%

**Comment by mikegfink** [comment]
@mikegfink%

**Comment by nicviclee** [comment]
:shipit:

**Comment by dorothy-ordogh** [comment]
@dorothy-ordogh%

**Comment by dorothy-ordogh** [comment]
:+1:

**Comment by dorothy-ordogh** [comment]
:shipit:

**Comment by mikegfink** [comment]
@rottencowz%

**Comment by rottencowz** [comment]
Approved :+1:

**Pull request merged**